src/App.tsx

```diff
@@ -14,7 +14,7 @@ function App() {
   }, []);
 
   function onClick() {
-    HelpScout.showNoty("success", "Hello from the sidebar app");
+    HelpScout.showNotification("SUCCESS", "Hello from the sidebar app");

```

Comment:
Is there a constant we can use here?

Reply:
Great call! Fixed in f3745452d3e6f06ffa83ea38de98d12f6eae92c3